danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerDSL.kt

```diff
@@ -8,13 +8,19 @@ data class DSL(
 )
 
 data class DangerDSL(
-        val github: GitHub?,
+        @Json(name ="github")
+        private val _github: GitHub?,
```

Comment:
@gianluz do you have a preference for the name of the private variable?

Reply:
well, in kotlin we don't use the `-` 😞 but if is something that we are following in all the danger projects happy to close one eye

Reply:
Yeah, it's a c and JS convention